server/website/website/views.py

```diff
@@ -1655,3 +1660,58 @@ def create_test_website(request):  # pylint: disable=unused-argument
     set_default_knobs(s4)
     response = HttpResponse("Success: create test website successfully")
     return response
+
+
+# For tuner status UI test
+@csrf_exempt
```

Comment:
Can you move this test into https://github.com/cmu-db/ottertune/blob/master/server/website/tests/test_views.py

Reply:
I think it's better to run this UI test together with integration tests?  This test is not only about *tuner_status_view* function. For example, there was a bug on celery tasks which makes the task view shows wrong task order.  You need to run background tasks and async tasks to reproduce such errors.